Src/Plugins/Tests.Security/Validators/SEC101_005.SlackTokenValidatorTests.cs

```diff
@@ -41,92 +41,124 @@ public void SlackTokenValidator_Test()
         [Fact]
         public void SlackTokenValidatorMockHttp_Test()
         {
-            var testCases = new[]
+            const string uri = "https://slack.com/api/auth.test";
+            const string token = "xoxb-1234";
+            string fingerprintText = $"[secret={token}]";
+            var dict = new Dictionary<string, string>
+                {
+                    { "token", token },
+                };
+
+            var requestWithToken = new HttpRequestMessage(HttpMethod.Post, uri);
+            requestWithToken.Content = new FormUrlEncodedContent(dict);
+
+
+            var testCases = new HttpMockTestCase[]
             {
-                new
+                new HttpMockTestCase
                 {
                     Title = "Testing Valid Token",
-                    HttpStatusCode = HttpStatusCode.OK,
+                    HttpStatusCodes = new List<HttpStatusCode>() { HttpStatusCode.OK },
+                    HttpRequestMessages = new List<HttpRequestMessage>() { requestWithToken },
                     // In all cases JSON is formatted in the order receieved from slack.
-                    HttpContent = new StringContent("{\"ok\": true," +
-                                                    "\"url\": \"testteam.slack.com\"," +
-                                                    "\"team\":\"test team\"," +
-                                                    "\"user\": \"testbot\"," +
-                                                    "\"team_id\": \"1234ABCD\"," +
-                                                    "\"user_id\": \"5678EFGH\"," +
-                                                    "\"bot_id\": \"0987ZYXV\"," +
-                                                    "\"is_enterprise_install\": false}",
-                                                    Encoding.UTF8,
-                                                    "application/json").As<HttpContent>(),
+                    HttpContents = new List<HttpContent>() {
+                                      new StringContent("{\"ok\": true," +
+                                                        "\"url\": \"testteam.slack.com\"," +
+                                                        "\"team\":\"test team\"," +
+                                                        "\"user\": \"testbot\"," +
+                                                        "\"team_id\": \"1234ABCD\"," +
+                                                        "\"user_id\": \"5678EFGH\"," +
+                                                        "\"bot_id\": \"0987ZYXV\"," +
+                                                        "\"is_enterprise_install\": false}",
+                                                        Encoding.UTF8,
```

Comment:
you could try to use something like:

@"{ ""json"": ""abc"" }"

with this, you won't need to add + and i think it will make easier to replace.